lib/config.h.in

```diff
@@ -72,8 +72,9 @@
 
 #define rm_log_perrorf(message, ...)                                                     \
     if(errno) {{                                                                         \
+        int _errsv = errno;                                                              \
```

Comment:
it might... see `NOTES` section of http://man7.org/linux/man-pages/man3/errno.3.html